src/traces/scattergeo/hover.js

```diff
@@ -0,0 +1,105 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+
+'use strict';
+
+var Fx = require('../../plots/cartesian/graph_interact');
+var Axes = require('../../plots/cartesian/axes');
+
+var getTraceColor = require('../scatter/get_trace_color');
+var attributes = require('./attributes');
+
+
+module.exports = function hoverPoints(pointData) {
+    var cd = pointData.cd,
+        trace = cd[0].trace,
+        xa = pointData.xa,
+        ya = pointData.ya,
+        geo = pointData.subplot;
+
+    if(cd[0].placeholder) return;
```

Comment:
Here's the comment you're looking for: https://github.com/plotly/plotly.js/pull/1004/files#diff-ad4f76ccd6044ed16514297078e13b84R1672

Putting comment about something where it is set is probably more robust than putting it where it is used.